home/.thymerc

```diff
@@ -0,0 +1,38 @@
+set :timer, 25*60
+set :timer_break, 5*60
+set :warning, 5*60
+set :warning_color, "red,bold"
+set :interval, 1
+set :tmux, true
+set :tmux_theme, "#[fg=mycolor,bg=mycolor]#[fg=%s]%s#[fg=mycolor,bg=mycolor]"
+
+option :t, :today, 'open today sheet' do
+  `vim -O ~/.thyme-today.md ~/.thyme-records.md < \`tty\` > \`tty\``
+end
+
+option :p, :break, 'start a break' do
+  run
+end
+
+option :b, :break, 'start a break' do
+  set :timer, 5*60
+  run
+end
+
+option :l, :long_break, 'start a long break' do
+  set :timer, 10*60
+  run
+end
+
+option :s, 'seconds num', 'run with custom seconds' do |num|
+  @timer = num.to_i
+  run
+end
+
+# before do
+#   `mplayer ~/music/flight-of-the-bumble-bee.mp3 &`
```

Comment:
:ship: it!

On Wed, May 13, 2015 at 10:50 AM, Tom Reznick <notifications@github.com>
wrote:

> In home/.thymerc
> <https://github.com/ContinuityControl/dotfiles/pull/16#discussion_r30238083>
> :
>
> > +  set :timer, 5*60
> > +  run
> > +end
> > +
> > +option :l, :long_break, 'start a long break' do
> > +  set :timer, 10*60
> > +  run
> > +end
> > +
> > +option :s, 'seconds num', 'run with custom seconds' do |num|
> > +  @timer = num.to_i
> > +  run
> > +end
> > +
> > +# before do
> > +#   `mplayer ~/music/flight-of-the-bumble-bee.mp3 &`
>
> This is awesome
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuityControl/dotfiles/pull/16/files#r30238083>.
>



-- 
Joel Nimety
Senior VP, Engineering
Continuity
203.285.8133 (office)
203.763.9263 (cell)
http://www.continuity.net
http://www.linkedin.com/in/jnimety

-- 


-- 

------------------------------
This email and all files transmitted with it are intended solely for use by 
the intended recipient.  Any other use is prohibited.  If you received this 
email in error, please notify the sender, delete the email and destroy all 
attachments.  Opinions and ideas expressed herein are solely those of the 
sender, and may not reflect the official position of Continuity.